src/js/components/Meter/utils.js

```diff
@@ -0,0 +1,18 @@
+import { colorForName } from '../../utils';
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
any reason why we are disabling this?

I think we should always added `export default { backgroundProps }`

Reply:
You are SO right.